### ISSUE TITLE
Combined Lookup Widget Layout Glitch

### ISSUE THREAD

**Issue by metas-mk** [metasfresh/metasfresh-webui-frontend#1641]
### Is this a bug or feature request?
Bug
### What is the current behavior?
Currently, the Lookup widget dropdowns seem sometimes to not be properly aligned with the corresponding field.
#### Which are the steps to reproduce?
- Open Sales Order window
- Record a new Sales Order for "Machine Parts"
- See Lookup dropdown in location/ User.
### What is the expected or desired behavior?
Shall be aligned under field and with same left border.

![image](https://user-images.githubusercontent.com/13365687/36845347-ab7c9300-1d56-11e8-9a1e-d74286f3daee.png)

**Comment by PabloSichert** [comment]
With #1685 the dropdown will be as wide as the actual input field.

It would look like this:

![image](https://user-images.githubusercontent.com/4450694/37572060-e60e33fe-2b05-11e8-93fa-91f513ec041f.png)

Is that alignment okay?

**Comment by metas-rc** [comment]
Results of IT


Tested on webui


* Currently, the allignment is like this:

![image](https://user-images.githubusercontent.com/16001442/37967958-894f4438-31d5-11e8-960e-37db5d0efa13.png)


Is this ok? 
@metas-mk  Please, move the task to done if you think it's ok.

**Comment by metas-mk** [comment]
Quick check locally. Shall be fixed now. Pls. check in IT again.

**Comment by metas-rc** [comment]
Results of IT

Tested on webui


* OK: Now the lookup widgets for sales order are perfectly alligned horizontally and vertically.

* Regression
* OK: In the orderline batchentry, the product and ifco are also alligned correctly
* OK: In the orderline advanced edit, all the fields that I checked are also alligned corrrectly.
* OK: The fields of sales order line look also alligned correctly if checked in grid view.
